### PR TITLE
rtpengine: fix compilation with iptables 1.8.8

### PR DESCRIPTION
**Pull request by PolynomialDivision**
The extension handling changed [0,1]. Fix compilation with iptables
1.8.8 [2].

[0] - https://git.netfilter.org/iptables/commit/?id=ef108943f69a6e20533d58823740d3f0534ea8ec
[1] - https://git.netfilter.org/iptables/commit/?id=6c689b639cf8e2aeced8685eca2915892d76ad86
[2] - https://github.com/openwrt/openwrt/pull/9886

### CODE REVIEW

File: net/rtpengine/patches/06-fix-iptables-extension-compilation.patch
Comment (BKPepe) [inline net/rtpengine/patches/06-fix-iptables-extension-compilation.patch:2]: Where is the commit message and commit description?
 - Who is the author?
  - Is this patch taken from upstream? If not, are you going to upstream this?

**Review by BKPepe — CHANGES_REQUESTED** [review]

File: net/rtpengine/patches/06-fix-iptables-extension-compilation.patch
Comment (PolynomialDivision) [reply net/rtpengine/patches/06-fix-iptables-extension-compilation.patch]: I sent it upstream:
https://github.com/sipwise/rtpengine/pull/1498

Comment (BKPepe) [reply net/rtpengine/patches/06-fix-iptables-extension-compilation.patch]: Now, it's better thanks! Hopefully, we don't need to maintain this patch forever and we will see what the upstream says about it.

**Comment by micmac1** [comment]
Let's wait for upstream approval. Thanks!

**Comment by PolynomialDivision** [comment]
> Let's wait for upstream approval. Thanks!

@micmac1  It is merged:
https://github.com/sipwise/rtpengine/pull/1498#event-6772445016

**Pull request merged**
